src/js/utils/DOM.js

```diff
@@ -65,6 +65,16 @@ export const containsFocus = (node) => {
   return !!element;
 };
 
+export const focusWithinDropPortal = (node) => {
```

Comment:
I don't see anything within this function that seems to have anything to do with "focus". It looks like this should be "withinDropPortal"?

Reply:
Renamed to `withinDropPortal`